packages/core/src/services/stellar/balance.js

```diff
@@ -6,6 +6,7 @@ export const assets = {
     process.env.MOBI_ASSET_ISSUER
   ),
   native: Asset.native(),
+  xlm: Asset.native(),
```

Comment:
Why do we need an extra native-XLM asset mapping here?

Reply:
In the application, we often use `xlm` as a key or label to reference the native asset. This addition will help to avoid mapping assets by hand every time.